### PR TITLE
Feature/update libraries

### PR DESCRIPTION
**Pull request by mfeckie**
This PR

- [x] Upgrades to Ember 3.16
- [x] Upgrades to broccoli-autoprefixer 7.x
- [x] Addresses changes to autoprefixer API
- [x] Adds changelog entry to note breaking change

I noticed that the changleog suggests that broccoli-autoprefixer is already at `7`, but this isn't reflected in `package.json`.

### CODE REVIEW

**Comment by snewcomer** [comment]
I really like this PR!  

@kimroen Do you have the bandwidth to review this PR.  If not, what do you think about transferring over to `adopted-ember-addons`?

**Comment by kimroen** [comment]
@mfeckie Thank you so much for making this PR, and sorry for the radio silence.

---

@snewcomer I also like it - thanks for poking me about it!

Unfortunately, as we've talked about directly elsewhere I don't see myself having the capacity to look at this or my other ember projects now or in the near future.

I've invited you (@snewcomer) to become a collaborator on this project and `ember-cli-document-title`, so if you can find the time and attention to give to these projects, please feel free to do so, and let's talk directly about access to publishing once there is something to publish.

Transferring over to `adopted-ember-addons` also feels like a good idea.

Thanks again!

**Comment by snewcomer** [comment]
@kimroen Really really appreciate it!  We are super grateful for these addons!  If possible, could I get npm publish rights as well in the meantime? `snewcomer`.  We can explore the adopted addons, but only with your explicit permission!

**Comment by mfeckie** [comment]
@snewcomer Have updated PR to bring it to Ember 3.16.  No other changes.

I'd be happy to see it merged, version bumped and released.

Also happy to become a maintainer should it become adopted.

File: .travis.yml [hunk 1]
```diff
@@ -1,35 +1,60 @@
...
+    - stage: "Additional Tests"
+      name: "Floating Dependencies"
+      install:
+        - npm install --no-package-lock
+      script:
+        - npm test
+
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.12
```
Comment (snewcomer) [inline .travis.yml]: Should we add 3.8 && even 3.4 for the time being?  It is such a large bump and likely if users haven't explored different solution with autoprefixer, they might be on older version?

Comment (mfeckie) [reply .travis.yml]: I've added 3.8.  3.4 doesn't work, `vendor.css` ends up empty.

I'm not sure it's reasonable to expect compatibility with 3.4.  Could we make this a 1.x release and state that we support 3.8 and above?  Folks on current versions can continue to use and folks that want updates (and are using 3.4) may need to do some work.

**Comment by kimroen** [comment]
> @kimroen Really really appreciate it! We are super grateful for these addons! If possible, could I get npm publish rights as well in the meantime? `snewcomer`. We can explore the adopted addons, but only with your explicit permission!

You now have publish rights to both packages :)

File: .travis.yml (hunk 1 above)
Comment (snewcomer) [reply .travis.yml]: Yep that is totally fine!  Was just ensuring we support the earliest versions that we could!  Thanks a ton!

**Comment by snewcomer** [comment]
@mfeckie Added to `1.0.0-beta.0`.  Our app (as is all of them :)) is very large and we have some CSS tweaks to work out it looks like as I tested this.  Other teams are verifying this so in the meantime put it in beta for a little bit until I get some feedback!  Thanks again for everything.

**Comment by bradleypriest** [comment]
This PR seems to have dropped compatibility with EmberCLI `config/targets.js` was that intentional, or an oversight?

**Comment by mfeckie** [comment]
It was intentional to bring the library in line with how autoprefixer expects things to be.

**Comment by bradleypriest** [comment]
In that case, it’s definitely worth adding a note in the changelog. 

And maybe a readme item on how to achieve the old logic if possible. 

I’d also wonder if you could reconsider?
 
It definitely breaks the principle of least surprise for me that an ember-cli addon based around browser support doesn’t automatically hook up the existing browser support config that ember-cli provides.

**Comment by mfeckie** [comment]
Happy to consider a PR with note for changelog and / or adding targets as an option again.

We did make the release a major to allow for breaking changes

**Comment by snewcomer** [comment]
ref #51 To add back support.  Lmk what you think!

**Pull request merged**